services/brig/test/integration/API/Provider.hs

```diff
@@ -689,12 +556,13 @@ testMessageBotTeam config crt db brig galley cannon = withTestService config crt
     let sid = sref^.serviceRefId
 
     -- Prepare user with client
```

Comment:
Traces of a slight refactoring I did regarding using `createUserWithTeam` in more places. We have to do whitelisting here if we want to be able to use the bot, and whitelisting can only be done by a team owner; `createUserWithTeam` ensures that the created user is an owner.